Packages/com.scene.handle.extensions/Handle Extensions/Editor/SceneHandles/SceneHandles.Radius2D.cs

```diff
@@ -27,346 +27,370 @@ static float Size2DSlider(int id, Vector3 center, Vector3 direction, Vector3 for
             size = UnityEditor.HandleUtility.GetHandleSize(position1);
-			position1 = Slider2DHandle(positionId1, position1, Vector3.zero, up, forward, right, size * 0.05f, OutlinedDotHandleCap);
-			if (GUI.changed)
-			{
-				var moveDelta = (position1 - center);
-				if (noRotation) radius1 = GeometryUtility.ProjectPointRay(center - moveDelta, center, rotatedDelta2);
-				else			radius1 = center - moveDelta;
-
-				delta1 = GeometryUtility.ProjectPointPlane(radius1 - center, plane);
-				delta1Magnitude = delta1.magnitude;
-
-				if (!noRotation && delta1Magnitude > Vector3.kEpsilon) radius2 = center + ((Quaternion.AngleAxis(-90, up) * (delta1 / delta1Magnitude)) * delta2Magnitude);
-				prevChanged = true;
-			}
-			
-			GUI.changed = false;
+            position1 = Slider2DHandle(positionId1, position1, Vector3.zero, up, forward, right, size * 0.05f, OutlinedDotHandleCap);
+            if (GUI.changed)
+            {
+                var moveDelta = (position1 - center);
+                if (noRotation) radius1 = GeometryUtility.ProjectPointRay(center - moveDelta, center, rotatedDelta2);
+                else			radius1 = center - moveDelta;
+
+                delta1 = GeometryUtility.ProjectPointPlane(radius1 - center, plane);
+                delta1Magnitude = delta1.magnitude;
+
+                if (!noRotation && delta1Magnitude > Vector3.kEpsilon) radius2 = center + ((Quaternion.AngleAxis(-90, up) * (delta1 / delta1Magnitude)) * delta2Magnitude);
+                prevChanged = true;
+
+                position0 = center + delta1;
+                position1 = center - delta1;
+                position2 = center + delta2;
+                position3 = center - delta2;
+            }
+            
+            GUI.changed = false;
             size = UnityEditor.HandleUtility.GetHandleSize(position2);
-			position2 = Slider2DHandle(positionId2, position2, Vector3.zero, up, forward, right, size * 0.05f, OutlinedDotHandleCap);
-			if (GUI.changed)
-			{
-				var moveDelta = (position2 - center);
-				if (noRotation) radius2 = GeometryUtility.ProjectPointRay(center + moveDelta, center, rotatedDelta1);
-				else			radius2 = center + moveDelta;
+            position2 = Slider2DHandle(positionId2, position2, Vector3.zero, up, forward, right, size * 0.05f, OutlinedDotHandleCap);
+            if (GUI.changed)
+            {
+                var moveDelta = (position2 - center);
+                if (noRotation) radius2 = GeometryUtility.ProjectPointRay(center + moveDelta, center, rotatedDelta1);
+                else			radius2 = center + moveDelta;
+
+                delta2 = GeometryUtility.ProjectPointPlane(radius2 - center, plane);
+                delta2Magnitude = delta2.magnitude;
 
-				delta2 = radius2 - center;
-				delta2Magnitude = delta2.magnitude;
+                if (!noRotation && delta2Magnitude > Vector3.kEpsilon) radius1 = center + ((Quaternion.AngleAxis(90, up) * (delta2 / delta2Magnitude)) * delta1Magnitude);
```

Comment:
Might be nicer as a code block instead of an in-line if statement, since there's a lot happening here

Reply:
good point